src/datamodule.py

```diff
@@ -106,6 +108,83 @@ def __len__(self):
         return len(self.chips_path)
 
 
+class ClayDatasetV1(Dataset, ClayDataset):
```

Comment:
Got `TypeError: Cannot create a consistent method resolution order (MRO) for bases Dataset, ClayDataset` when trying to instantiate this class. Think we should just subclass either `Dataset` or `ClayDataset`, not both.